Codigo/ScenarioDeathMatch.cls

```diff
@@ -465,8 +465,8 @@ Private Sub RespawnItems(ByVal frametime As Long)
     Dim SpawnCount As Integer
     For i = 0 To UBound(ItemsToSpawn) - 1
         If UpdateTime(ItemsToSpawn(i).timer, frametime) Then
-            For j = 0 To ItemsToSpawn(i).StackCount - 1
-                SpawnCount = ItemsToSpawn(i).itemCount * GenericGlobalLobby.RegisteredPlayers
+            For j = 0 To (ItemsToSpawn(i).StackCount * GenericGlobalLobby.RegisteredPlayers) - 1
```

Comment:
`(ItemsToSpawn(i).StackCount * GenericGlobalLobby.RegisteredPlayers) - 1` I would move this to a temp variable in the loop and add an error handler